src/music.ts

```diff
@@ -168,18 +169,18 @@ export class Music {
     };
   }
 
-  isValidNoteValue(note) {
+  isValidNoteValue(note: number): boolean {
     if (note == null || note < 0 || note >= Music.NUM_TONES) {
       return false;
     }
     return true;
   }
 
-  isValidIntervalValue(interval) {
+  isValidIntervalValue(interval: number): boolean {
```

Comment:
Using the comparison from note will not allow the interval `octave`

Reply:
I suggest to remove this function and use the approach from `getIntervalValue` on the places where this function is used

Reply:
@rvilarl this should be a bug ticket. It does not belong to this PR